components/bme280/src/bme280_defs.h

```diff
@@ -10,6 +11,14 @@
  * @brief This file contains definitions for the BME280 driver.
  */
 
+/**
+ * @brief Configuration structure for BQ27441.
```

Comment:
So is this for BME280 or BQ27441?

Reply:
Youre right, copy and paste got me. Fixed